routes/homeRoutes.js

```diff
@@ -12,6 +12,6 @@ router.post("/signup", authController.postSignup);
 router.get("/signup", authController.getSignup);
 router.get("/login", authController.getLogin);
 router.post("/login", authController.postLogin);
-router.post("/logout", authController.logout);
+router.get("/logout", authController.logout);

```

Comment:
shouldn't this be a post?

Reply:
Fetch uses ajax so it doesn't allow you to redirect through the backend. We could redirect client side, but not sure what the team wants in how to redirect the user.

Reply:
An issue for this has been created and will be resolved later.